README.md

```diff
@@ -20,12 +20,18 @@ Any additional arguments supported with `docker build` are supported
 sm-docker build . --file /path/to/Dockerfile --build-arg foo=bar
 ```
 
-By default, the image will be pushed to a repository `sagemakerstudio` with the tag `latest`, and use the Studio App's execution role and the default SageMaker Python SDK S3 bucket
+By default, the CodeBuild project will not run within a VPC, the image will be pushed to a repository `sagemakerstudio` with the tag `latest`, and use the Studio App's execution role and the default SageMaker Python SDK S3 bucket
 
 These can be overridden with the relevant CLI options.
 
 ```bash
-sm-docker build . --repository mynewrepo:1.0 --role MyRoleName 
+sm-docker build . --repository mynewrepo:1.0 --role MyRoleName --bucket MyBucketName --vpc-id MyVpcId --subnets MySubnetId1,MySubnetId2 --security-groups MySecurityGroup1,MySecurityGroup2
```

Comment:
```suggestion
# Basic usage
sm-docker build .

# Specify a custom ECR repository to push to and custom role
sm-docker build . --repository mynewrepo:1.0 --role MyRoleName

# Connect the CodeBuild job to a VPC
sm-docker build . --repository mynewrepo:1.0 --role MyRoleName --bucket MyBucketName --vpc-id MyVpcId --subnets MySubnetId1,MySubnetId2 --security-groups MySecurityGroup1,MySecurityGroup2
```

Reply:
My recommendation is that we keep the basic example as-is,  but add more "advanced" usage examples like using VPC.

Reply:
Addressed as part of the change recommended on the second comment below, where the suggestion was made to use the extended example with sample values in place of the earlier sample example